src/Components/Endpoints/src/DependencyInjection/WebAssemblyComponentsEndpointOptions.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Components.Endpoints;
+
+/// <summary>
+/// Options to configure interactive WebAssembly components in a razor components application.
+/// </summary>
+public sealed class WebAssemblyComponentsEndpointOptions
+{
+    /// <summary>
+    /// Gets or sets <see cref="PathString"/> that indicates the prefix for Blazor WebAssembly framework files.
+    /// </summary>
+    public PathString FrameworkFilesPathPrefix { get; set; }
+}
```

Comment:
This API allows the path prefix passed to `UseBlazorFrameworkFiles` to be configurable in a "full-stack Blazor"-style app.

Reply:
Is this something we could avoid?

Reply:
What's the reason for wanting to avoid it? Is it to keep the API smaller for the initial release? Without this API, I don't think there's a way for customers to configure the path prefix passed to `UseBlazorFrameworkFiles()`. I was using this API in our own test app so that `Components.TestServer` could reference both `BasicTestApp` as well as a new, minimal WebAssembly project.

Reply:
Seems reasonable to allow this configurability if it's what's need to host multiple independent WebAssembly apps on a single server app.

Reply:
This only impacts the path from which `_framework` folder content is loaded. The paths from which `wwwroot` content gets served can be configured with `UseStaticFiles()`.

I'm worried `PathPrefix` might not be clear enough in the context of calling `AddWebAssemblyComponents()`. I think `StaticAssetsPathPrefix` is a bit too general given that it's specifically `_framework` files that we're configuring.

Although, I can't think of a case where the `_framework` path prefix would be different than the path prefix for `wwwroot` content. So maybe it's okay to go with `StaticAssetsPathPrefix`? But maybe then there could still be some confusion around whether calling `UseStaticFiles()` is still necessary.

Perhaps we keep the current name but update the XML comment to make this clearer?

Reply:
It also affects `_content`, doesn't it? Or if not I think that would be at risk of clashing. And possibly any other framework-supplied files such as hot reload scripts, though I'm not so sure about that.

I don't feel strongly about the name, so if you think `FrameworkFilesPathPrefix` is the safest choice in terms of accuracy and understandability, that's fine with me.

Reply:
Let's do `PathPrefix` for now - one could make the argument that the fact that it only affects `_framework` files is an uninteresting detail to the developer.

Reply:
> I could totally be missing something, but from what I can tell it looks like _content isn't affected by this:

Is that a bug then? I thought the scenario here was "hosting unrelated WebAssembly apps on separate paths". If that's the case we can't have them both using `/_content`.